dashboard/test/controllers/pd/workshop_daily_survey_controller_test.rb

```diff
@@ -1029,6 +1029,77 @@ class WorkshopDailySurveyControllerTest < ActionDispatch::IntegrationTest
       assert_select 'h1', text: 'Thank you for submitting today’s survey.'
     end
 
+    test 'AYW1 pre-survey link shows foorm survey' do
+      setup_academic_year_workshop
+      sign_in @enrolled_academic_year_teacher
+
+      pre_survey_links = %w(/pd/AYW1/pre /pd/AYW1/pre/module/1 /pd/AYW1/pre/module/2)
+      pre_survey_links.each do |link|
+        get link
+        assert_response :success
+        assert_template :new_general_foorm
+      end
+    end
+
+    test 'AYW1 post-survey link shows foorm survey for attended teacher' do
+      setup_academic_year_workshop
+      sign_in @enrolled_academic_year_teacher
+      create :pd_attendance,
+        session: @academic_year_workshop.sessions[0],
+        teacher: @enrolled_academic_year_teacher,
+        enrollment: @academic_year_enrollment
+
+      pre_survey_links = %w(/pd/AYW1/post/in_person /pd/AYW1/post/module/1 /pd/AYW1/post/module/2 /pd/AYW1/pre/module/1_2)
```

Comment:
Typo variable name? Also the last case is a pre-survey?

Reply:
ah yeah going too fast on this one :)